Amazon Alexa Reviews Analysis/Model/Amazon_Alexa_Reviews_Analysis.py

```diff
@@ -0,0 +1,101 @@
+# Amazon Alexa Reviews Analysis
```

Comment:
shift this file to the Amazon Alexa Reviews Analysis folder. Also, rename this file to training.py